### ISSUE TITLE
check if caching server is reachable before using

### ISSUE THREAD

**Issue by dnikles** [ninxsoft/mist-cli#161]
### :bulb: Description

If a caching server is given but is unreachable, Mist will fail to download the installer.

### :white_check_mark: Proposed solution

Check if a given caching server is reachable, and failover to not using the caching server if it is not.

### :adhesive_bandage: Alternatives Solutions + Workarounds

Currently this can be accomplished with a script outside of Mist, then calling mist-cli with the appropriate option (or lack thereof)

### :information_source: Additional context

- Would you also like to see this implemented in [Mist.app](https://github.com/ninxsoft/Mist)?:x:

**Comment by ninxsoft** [comment]
Makes sense! I'll see how much work is involved to default to no caching server, if the specified one is not accessible 👍
